packages/epics/src/agreements/plugins/payment-schedule/payment-schedule.tsx

```
@@ -0,0 +1,215 @@
+} from './validation';
+
+const options = ['Immediately', 'Future Payment', 'Milestones'] as const;
+type Option = (typeof options)[number];
+
+export type FormValues = {
+  option: Option;
+  futureDate?: Date;
+  milestones: Milestone[];
+};
+
+type PaymentScheduleProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)